src/messages/store/publish.ts

```diff
@@ -168,7 +173,7 @@ type SignAndBroadcastConfiguration = {
     sync: boolean;
 };
 
-async function sendMessage(configuration: SignAndBroadcastConfiguration, file: Buffer | Blob | File): Promise<any> {
```

Comment:
Why did you remove "Buffer" from the possible value? It's the default value when loading a file in memory on Node.js

Reply:
Because raw Buffers cannot be sent in a FormData format to the server. They either have to be Blobs or Files. Everything that could be done with a buffer (computing the hash) has already been done by this point.

Reply:
And for that, users should use `Publish` instead. But yeah, it might be a bit better constructed. Up until now, the SDK was not compatible with Node.js, because `utils.ts` was using `FileReader`

Reply:
The tests I have done led to the conclusion that sending Buffers was supported (that's why I could assess that Node.js worked while the browser upload was crashing). 

I had an issue with V8 Garbage Collector while trying to convert a Buffer to a Blob in Node.js. This is because it allocates a new memory slot to wrap the Buffer. I don't think that using Blob as the default format is optimal if we can already use the Buffer.

Reply:
But as I described, I _have to_ wrap it in a Blob to be able to pass it into FormData :(